app/src/main/java/com/poklad/androidifystore/data/remote/StoreApi.kt

```diff
@@ -0,0 +1,30 @@
+package com.poklad.androidifystore.data.remote
+
+import com.poklad.androidifystore.data.remote.model.ProductItemResponse
+import com.poklad.androidifystore.utils.ApiConstants.ALL_PRODUCTS
```

Comment:
don't import constants without class Prefix. It's harder to read, as you need to check imports to understand where this const was declared